playground/utils.ts

```diff
@@ -56,3 +66,73 @@ export async function transfer(
 
   return await publicClient.waitForTransactionReceipt({ hash })
 }
+
+export async function setup4337Playground(
+  safe4337Pack: Safe4337Pack,
+  {
+    nativeTokenAmount,
+    erc20TokenAmount,
+    erc20TokenContractAddress
+  }: {
+    nativeTokenAmount?: bigint
+    erc20TokenAmount?: bigint
+    erc20TokenContractAddress: string
+  } = {
+    erc20TokenAmount: 200_000n,
+    erc20TokenContractAddress: '0xFC3e86566895Fb007c6A0d3809eb2827DF94F751'
+  }
+): Promise<{ transactions: MetaTransactionData[]; timestamp: bigint }> {
+  const senderAddress = await safe4337Pack.protocolKit.getAddress()
+
+  // Log supported entry points and Safe state
+  console.log('Supported Entry Points', await safe4337Pack.getSupportedEntryPoints())
+  console.log('Chain id', await safe4337Pack.getChainId())
+  console.log('Safe Address: ', senderAddress)
+  console.log('Safe Owners:', await safe4337Pack.protocolKit.getOwners())
+  console.log('is Safe Account deployed: ', await safe4337Pack.protocolKit.isSafeDeployed())
+
+  const externalProvider = safe4337Pack.protocolKit.getSafeProvider().getExternalProvider()
+  const externalSigner = await safe4337Pack.protocolKit.getSafeProvider().getExternalSigner()
+  const signerAddress = await safe4337Pack.protocolKit.getSafeProvider().getSignerAddress()
+
+  if (!externalSigner || !signerAddress) {
+    throw new Error('No signer found!')
+  }
+
+  // Fund Safe
+  if (nativeTokenAmount) {
+    console.log(`sending ${formatEther(nativeTokenAmount)} native tokens...`)
+
+    const hash = await externalSigner?.sendTransaction({
+      to: senderAddress,
+      value: nativeTokenAmount,
+      chain: sepolia
```

Comment:
This sets up 4337 playground but is not getting the chain from the .env-sample that is inside relay-kit folder. This may cause issues if someone intends to use a chain different than sepolia.